azure-pipelines.yml

```diff
@@ -68,6 +68,38 @@ stages:
       value: $(Build.SourceBranchName)
 
   jobs:
+  - job: get_impacted_area
+    displayName: "Get impacted area"
+    timeoutInMinutes: 240
+    continueOnError: true
+#    pool: sonic-ubuntu-1c
+    steps:
+      - template: .azure-pipelines/impacted_area_testing/get-impacted-area.yml
+
+  - job: impacted_area_t0_elastictest
+    displayName: "kvmtest-t0-on-needed by Elastictest"
+    dependsOn: get_impacted_area
+    condition: or(contains(dependencies.get_impacted_area.outputs['SetVariableTask.PR_CHECKERS'], 't0'), eq(dependencies.get_impacted_area.outputs['SetVariableTask.PR_CHECKERS'], ''))
```

Comment:
The test jobs rely heavily on the previous job, so the get_impacted_area job must be robust enough to return the necessary variables at all times.

Reply:
Yeah, test job depends on the job `Get impacted area`. If the job `Get impacted area` fails, other jobs will not start.